umbra-js/src/classes/Umbra.ts

```diff
@@ -819,7 +819,7 @@ export async function assertSufficientBalance(signer: JsonRpcSigner | Wallet, to
   // If applicable, check that sender has sufficient token balance. ETH balance is checked on send. The isEth
   // method also serves to validate the token input
   if (!isEth(token)) {
-    const tokenContract = new Contract(token, ERC20_ABI, signer) as ERC20;
+    const tokenContract = new Contract(token, ERC20_ABI, signer) as unknown as ERC20;
```

Comment:
Like the above use the factory class.

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/577/commits/92de58d332ad2e4814a9569b068bdca6524da124

Reply:
Yes

Reply:
Thxxx!